src/objective-c/examples/SampleCpp/SampleCpp/ViewController.m

```diff
@@ -0,0 +1,54 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#import "ViewController.h"
+
+#import <Cronet/Cronet.h>
+#import <RemoteTestCpp/messages.grpc.pb.h>
+#import <RemoteTestCpp/test.grpc.pb.h>
+#import <grpc/grpc_cronet.h>
+#import <grpcpp/create_channel.h>
+#import <grpcpp/impl/codegen/client_context.h>
+#import <grpcpp/security/credentials.h>
+
+@implementation ViewController
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  [Cronet setHttp2Enabled:YES];
```

Comment:
Do we still have to use ObjC API for Cronet?

Reply:
We can use the C API if required, but I haven't seen a use case that requires it.

Reply:
Couple things need to be done if we choose to migrate to Cronet C API:
1. Need to update Cronet.Framework to a newer version that supports the C API
2. Cronet needs to provide a C API to mock the certificate verifier similar to Objective-C API `enableTestCertVerifierForTesting`